controllers/articles.controllers.js

```diff
@@ -20,7 +20,17 @@ exports.getArticles = async (req, res, next) => {
         res.status(200).send({articles: articles})
     }
     catch(err) {
-        console.log(err)
         next(err)
     }
 }
+
+exports.getArticleComments = async (req, res, next) => {
+    try {
```

Comment:
Here you need to check if the article ID exists, otherwise how would you tell if the article doesn't exist or just no comments for the article?